docs/content/getting_started/build_libmesh.md

```diff
@@ -0,0 +1,20 @@
+#### Compile libMesh
+* MOOSE directly relies on the libMesh finite-element framework. Because of this strong tie MOOSE contains a particular version of libMesh that we have vetted for our users. We will periodically update that version (about once a month) so stay tuned to the MOOSE-users mailing list for those announcements. To pull down and compile this version of libMesh you simply need to run a script in MOOSE:
+
+    <pre>
+    cd ~/projects/moose
+    scripts/update_and_rebuild_libmesh.sh
+    </pre>
+
+!!! warning "Do not use `sudo` when running update_and_rebuild_libmesh.sh."
```

Comment:
Did you want the "title" to be "Do not use..." Generally, these should be written as:

!!! warning
    Do not use `sudo` when ...